cmd/injector/main.go

```diff
@@ -52,12 +52,6 @@ func main() {
 		log.Fatalf("Error set env: %v", err)
 	}
 
-	// Initialize dapr metrics exporter
-	err = metricsExporter.Init()
-	if err != nil {
-		log.Fatal(err)
-	}
-
```

Comment:
`metricsExporter.Init` renamed to `metricsExporter.Run`, and is a now a long running process which closes on context cancellation.